docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,53 @@
+# Access Control
+## AC-05 - Separation of Duties
+
+The organization:  
+a. Separates [at a minimum, data creation and control, software development and maintenance, and security functions];
+
+b. Documents separation of duties of individuals; and 
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+The TANF Data Portal (TDP) system has explicitly defined the roles and responsibilities that accounts perform to operate and maintain the system. This includes data creation and processing, software development, maintenance, and security implementation.
+
+Every role has flags associated with the account.  These flags are *SuperUser* and *Staff* and are set to *False* by default.  When a new TDP account is created, the OFA Admin has the ability to set the flags to *True* based on the user's job responsibilities.  Note: if an account has the *Staff* flag set to *True*, the account will have access to Django Admin, but will not have access to any of the account management functionality.
+
+The TDP system has two roles: OFA Admin and Data Prepper.
+  * OFA Admin has the following abilities: security functionality to manage TDP accounts through Django Admin, upload data on behalf of Data Preppers, upload data files locally into the web application
+  * OFA Admin *SuperUser* and *Staff* flags are both set to *True*.  This gives the OFA Admin the ability to access Django Admin and access to account management functionality.
+  * Data Prepper roles are users from the states, tribes, and territories (STT) who will be uploading data.  Data Preppers collect TANF data, create data files, and transmit final data to OFA through the TANF application.  Data Preppers reads data over assigned region and are able to upload and submit new TANF reports, replace and resubmit TANF reports.
+  * Data Prepper *SuperUser* and *Staff* flags are both set to *False*.  Data Preppers do not have access to Django Admin, cannot provision new TDP accounts, and do not have access to the account management functionality.
+
+Developers of the TDP application are responsible for the software development and maintenance of the system.
+  * Developers are granted only enough permission in GitHub and Cloud.gov to support their duties.    
+  * All code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.
+
+The System Owner would have *Staff* and *SuperUser* access along with the OFA Admin role and has the ability to manage TDP accounts, including provisioning new user accounts on request. Other users with the role OFA Admin would not have *Staff* and *SuperUser* which separates the System Owner from OFA Admin.  
+
+
+**Part b.**  
+
+**Developers**  
+Developers are granted only enough permission in GitHub and Cloud.gov in order to support their duties.  All code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.  
```

Comment:
`Product Manager` is not a role we've defined, I believe this should be `Product Owner`.

Reply:
updated to Product Owner

Reply:
Looking at this again...should this be System Owner?  I think this is the first time I introduce Product Owner.